app/gather/api/views.py

```diff
@@ -45,3 +48,8 @@ class MaskViewSet(MtViewSetMixin, ModelViewSet):
     search_fields = ('survey__name', 'name', 'columns',)
     ordering = ('survey', 'name',)
     mt_field = 'survey'
+
+
+@api_view(['GET'])
+def get_es_consumer_url(request, *args, **kwargs):
```

Comment:
for this kind of things there is a view in `gather/views.py`  `assets_settings` to return the url.